app/assets/javascripts/discourse/templates/components/pwa-install-banner.hbs

```diff
@@ -0,0 +1,10 @@
+{{#if showPWAInstallBanner}}
+  <div class="row">
+    <div class="pwa-install-banner alert alert-info">
+      <div class="close" {{action "dismiss"}}>
+        {{d-icon 'times'}}
+      </div>
+      {{i18n 'pwa.install_banner.question'}} <a {{action "turnon"}}>{{i18n 'pwa.install_banner.link' title=siteSettings.title}}</a>.
```

Comment:
Can we wrap `{{i18n 'pwa.install_banner.question'}}` in a span please? I don't like text with no direct html parent.